docs/monitoring.md

```diff
@@ -479,11 +479,17 @@ can be identified by their `[attempt-id]`. In the API listed below, when running
     <td><code>/applications/[app-id]/stages/[stage-id]</code></td>
     <td>
       A list of all attempts for the given stage.
+      <br><code>?details=true</code> list all attempts with stage detail data.
+      <br><code>?status=[active|complete|pending|failed]</code> list only stages in the state, only effect when <code>details=true</code>..
```

Comment:
<br><code>?taskStatus=[RUNNING|SUCCESS|FAILED|KILLED|PENDING]</code> lists only those tasks in the specified task status. Query parameter taskStatus takes effect only when <code>details=true</code>.

Reply:
Done